client/client.go

```diff
@@ -282,15 +250,28 @@ func (c *Client) withAccountIDRegionAndNamespace(accountID, region, namespace st
 	}
 }
 
+func isValidRegions(regions []string) error {
+	// validate regions values
+	for i, region := range regions {
+		if i != 0 && region == "*" {
+			return fmt.Errorf("region wildcard \"*\" is only supported as first argument")
+		}
+	}
+	return nil
+}
+func isAllRegions(regions []string) bool {
+	wildcardAllRegions := false
+	if (len(regions) == 1) && (regions[0] == "*" || len(regions) == 0) {
```

Comment:
I think this should be `(len(regions) == 1 && regions[0] == "*") || len(regions) == 0`

Reply:
you are right. Caught it in the test cases